rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -123,7 +123,7 @@ rocalJpegFileSourceSingleShard(
         unsigned max_height,
         RocalDecoderType dec_type)
 {
-    Image* output = nullptr;
+    rocalTensor* output = nullptr;
     auto context = static_cast<Context*>(p_context);
     try
```

Comment:
can we please add the condition check to verify valid context is passed / not?
if (p_context == nullptr)) {
        ERR("Invalid ROCAL context or invalid input image")
        return output;
    }

Reply:
Have we added this check ?